packages/round-manager/src/features/round/ApplicationFormComponents.tsx

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Instead of doing this on the whole file, would it be better to be explicit on which lines we are applying this lint exclusion to?

Reply:
I do not think so - the anys are necessary in this case for each component. See: https://github.com/gitcoinco/grants-stack/pull/1805#issuecomment-1551915471

Reply:
@hmrtn you can remove the anys like this:

```diff
diff --git a/packages/round-manager/src/features/round/ApplicationFormComponents.tsx b/packages/round-manager/src/features/round/ApplicationFormComponents.tsx
index 631233ce..92de79f3 100644
--- a/packages/round-manager/src/features/round/ApplicationFormComponents.tsx
+++ b/packages/round-manager/src/features/round/ApplicationFormComponents.tsx
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 import { FC } from "react";
 import "react-datetime/css/react-datetime.css";
 import {
@@ -7,6 +6,7 @@ import {
   FieldErrors,
   useController,
   UseFormRegisterReturn,
+  Path,
 } from "react-hook-form";
 import { Listbox, RadioGroup } from "@headlessui/react";
 import { Program, Round } from "../api/types";
@@ -15,15 +15,17 @@ import moment from "moment";
 import { FormInputField } from "../common/FormInputField";
 import { FormDropDown } from "../common/FormDropDown";
 
-export function RoundName(
-  props: any
-) {
+export function RoundName(props: {
+  register: UseFormRegisterReturn<string>;
+  errors: FieldErrors<Round>;
+}) {
   return (
     <FormInputField
       {...props}
       label="Round Name"
       id="roundMetadata.name"
-      placeholder="Enter round name here."    />
+      placeholder="Enter round name here."
+    />
   );
 }
 
@@ -61,7 +63,10 @@ export function ProgramChain(props: { program: Program }) {
   );
 }
 
-export const ContactInformation = (props: any) => (
+export const ContactInformation = (props: {
+  register: UseFormRegisterReturn<string>;
+  errors: FieldErrors<Round>;
+}) => (
   <FormInputField
     {...props}
     label="Contact Information"
@@ -83,7 +88,7 @@ export function Support(props: {
     "Telegram Group Invite Link",
     "Google Form Link",
     "Other (please provide a link)",
-  ]
+  ];
 
   return (
     <FormDropDown
@@ -181,36 +186,40 @@ export function RoundType(props: {
   );
 }
 
-
 interface DatetimeProps {
   control: Control<Round>;
-  name: any;
+  name: Path<Round>;
   label: string;
   date?: moment.Moment;
-  setDate?: (date: moment.Moment) => void;  
+  setDate?: (date: moment.Moment) => void;
   minDate?: moment.Moment;
 }
 
-export const Datetime: FC<DatetimeProps> = ({ control, name, label, date, setDate, minDate }) => {
-  const now = moment().format('YYYY-MM-DDTHH:mm');
+export const Datetime: FC<DatetimeProps> = ({
+  control,
+  name,
+  label,
+  date,
+  setDate,
+  minDate,
+}) => {
+  const now = moment().format("YYYY-MM-DDTHH:mm");
   return (
     <div className="relative w-full border-0 p-0 placeholder-grey-40 focus:ring-0 text-sm">
-      <label
-        htmlFor={name}
-        className="block text-[10px]"
-      >
+      <label htmlFor={name} className="block text-[10px]">
         {label}
       </label>
       <Controller
         control={control}
         name={name}
-        defaultValue={date?.format('YYYY-MM-DDTHH:mm')}
+        defaultValue={date?.format("YYYY-MM-DDTHH:mm")}
         render={({ field }) => (
           <input
-            data-testid={`${name}-testid`} 
+            data-testid={`${name}-testid`}
             type="datetime-local"
             {...field}
-            min={minDate ? minDate.format('YYYY-MM-DDTHH:mm') : now}
+            value={field.value?.toString()}
+            min={minDate ? minDate.format("YYYY-MM-DDTHH:mm") : now}
             className="block w-full border-0 p-0 focus:ring-0"
             onChange={(e) => {
               // Convert the local datetime to UTC
@@ -224,4 +233,3 @@ export const Datetime: FC<DatetimeProps> = ({ control, name, label, date, setDat
     </div>
   );
 };
-
```

Reply:
I also noticed you might not be using prettier, do you mind running it

Reply:
Resolved in #1656!